pkg/webhooks/group/group.go

```diff
@@ -114,11 +117,28 @@ func (s *GroupWebhook) authorized(request admissionctl.Request) admissionctl.Res
 				return ret
 			}
 		}
+
+		log.Info("Denying access", "request", request.AdmissionRequest)
+		ret = admissionctl.Denied(fmt.Sprintf("Prevented from accessing Red Hat managed Admin groups. Customer user groups may NOT match this regular expression: %s", protectedAdminGroups))
```

Comment:
```suggestion
		ret = admissionctl.Denied(fmt.Sprintf("Prevented from accessing Red Hat managed Admin groups. Customers should use https://cloud.redhat.com/openshift to manage groups that match this regular expression: %s", protectedAdminGroups))
```

This is a bit of a modification to the proposed message from the spreadsheet. @lisa will probably want to vet the specific messages

Reply:
👍  to Candace's suggested change here.